Shield.Core/Manager.cs

```diff
@@ -153,6 +154,7 @@ private void OnCharReceived(char c)
             this.buffer.Append(c);
             if (isComplete)
             {
+                Debug.WriteLine(string.Empty);
```

Comment:
What effect does writing an empty string have? Is there actually something that should be written here?